### PR TITLE
Add methods from 10.5

### PR DESCRIPTION
**Pull request by gcasa**
Add missing 10.5 methods.

### CODE REVIEW

File: Headers/Foundation/NSRunLoop.h [hunk 1]
```diff
@@ -34,12 +34,18 @@ extern "C" {
 
 @class NSTimer, NSDate, NSPort;
 
+typedef NSString* NSRunLoopMode;
```
Comment (fredkiefer) [inline Headers/Foundation/NSRunLoop.h:37]: Maybe we should use this new type for all the cases where mode is used here (and in the long run all the places where we call a run loop).

File: Source/NSURLConnection.m [hunk 2]
```diff
@@ -157,13 +157,37 @@ + (NSURLConnection *) connectionWithRequest: (NSURLRequest *)request
...
+  NSArray *modes = [NSArray arrayWithObject: mode];
+  [aRunLoop performSelector: @selector(start)
+                     target: self
+                   argument: nil
+                      order: 0
+                      modes: modes];
+}
+
+- (void) unscheduleFromRunLoop: (NSRunLoop *)aRunLoop 
+                       forMode: (NSRunLoopMode)mode
```
Comment (fredkiefer) [inline Source/NSURLConnection.m:184]: It looks wrong that the mode argument gets ignored, but I see no way of using it.

**Review by fredkiefer — APPROVED** [review]
Looks good to me.

File: Headers/Foundation/NSRunLoop.h (hunk 1 above)
Comment (gcasa) [reply Headers/Foundation/NSRunLoop.h:37]: Okay, I will make this change.

File: Source/NSURLConnection.m (hunk 2 above)
Comment (gcasa) [reply Source/NSURLConnection.m:184]: That was my confusion as well.

**Pull request merged**